modal/_utils/grpc_utils.py

```diff
@@ -173,6 +173,14 @@ async def retry_transient_errors(
                 # no point sleeping if that's going to push us past the deadline
                 raise exc
 
+            if isinstance(exc, AttributeError):
+                # StreamTerminatedError are not properly raised in grpclib<=0.4.7
+                # fixed in https://github.com/vmagamedov/grpclib/issues/185
+                # TODO: update to newer version (>=0.4.8) once stable
+                if "_write_appdata" in str(exc):
+                    continue
```

Comment:
`continue` isn't right here, because it will also skip the `n_retries` increment etc.

Reply:
ah yeah, fixed